src/libs/actions/Session.js

```diff
@@ -420,6 +441,76 @@ function validateEmail(accountID, validateCode, password) {
         });
 }
 
+// It's necessary to throttle requests to reauthenticate since calling this multiple times will cause Pusher to
+// reconnect each time when we only need to reconnect once. This way, if an authToken is expired and we try to
+// subscribe to a bunch of channels at once we will only reauthenticate and force reconnect Pusher once.
+const throttledReauthenticatePusher = _.throttle(() => {
```

Comment:
```suggestion
const throttledReauthenticatePusher = _.throttle(reauthenticatePusher, 5000, {trailing: false})
```

Reply:
Also, should `reauthenticatePusher` be declared as a `const` so that it's hoisted?

Reply:
Ah no that's backwards `function something() {}` will get hoisted `const` doesn't